tests/network.py

```diff
@@ -11,6 +12,18 @@
 
 BRIDGE_NF_CALL_IPTABLES = "/proc/sys/net/bridge/bridge-nf-call-iptables"
 
+
+@contextmanager
+def managed_nspopen(*args, **kwds):
+    # Code to acquire resource, e.g.:
+    proc = NSPopen(*args, **kwds)
+    try:
+        yield proc
+    finally:
+        # Code to release resource, e.g.:
+        proc.release()
```

Comment:
This "e.g." looks like a leftover from a copy paste. Also I think we need to `proc.wait()` before we `proc.release` given the NSPopen API.

Reply:
> This "e.g." looks like a leftover from a copy paste. 
Yes it is

> Also I think we need to proc.wait() before we proc.release given the NSPopen API.

I'll take a look if it's necessary. It already fixed the out of memory issues I was haveing without it.